javascript/javascript1/week2/script_2.js

```diff
@@ -0,0 +1,82 @@
+//*EVENT APPLICATION
+
+let todayDay = new Date();
+
+let getEventWeekday = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+
+function dateEvent(eventDay) {
+  let eventDate = new Date(todayDay);
+  eventDate.setDate(todayDay.getDate() + eventDay);
+
+  let eventWeekday = eventDate.getDay();
+
+  return getEventWeekday[eventWeekday];
```

Comment:
Here I would attempt this part again but using the [modulo function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Remainder) (also known as remainder function)

```
function getEventWeekday(eventDay) {
   const today = new Date();
   const weekday = today.getDay();
   // Find the weekday index in the weekdays array
   // apply the modulo operator
   // return the result
}
```